bindinfo/bind_cache_test.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2022 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bindinfo
+
+import (
+	"strconv"
+	"strings"
+	"testing"
+
+	"github.com/pingcap/tidb/util/hack"
+	"github.com/pingcap/tidb/util/mock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestBindCache(t *testing.T) {
```

Comment:
Please add one more test to check whether it can trace memory usage correctly.

Reply:
I will add more tests about this in the next [PR](https://github.com/pingcap/tidb/pull/32567).